thunder/tests/test_nvfuser.py

```diff
@@ -991,3 +991,78 @@ def make_integer_tensor():
         rout = f(x.cpu(), y.cpu()).to(device)
         jout = f(x, y)
         assert rout.equal(jout)
+
+
+@instantiate(
+    dtypes=(thunder.float16, thunder.bfloat16),
+    devicetypes=(devices.DeviceType.CUDA,),
+    executors=(nvFuserExecutor,),
+    decorators=(
+        pytest.mark.skipif(
```

Comment:
To streamline our internal QA process, every test that uses Flash Attention should be skipped on older GPUs using the compute capability check (https://github.com/Lightning-AI/lightning-thunder/pull/1327).

Reply:
Thanks, @IvanYashchuk! Is there a way to avoid having to remember this? E.g. let CI run tests on older GPUs because apparently people still care about them.

Reply:
Currently, Thunder's CI uses Lightning's resources. We could add runners with NVIDIA's resources including older GPUs similarly to how nvFuser's CI is set up. I would postpone doing that until we hit the problem again.

Alternatively, the test (and any test that uses Flash Attention) can be rewritten with a parametrization that turns on and off Flash Attention in PyTorch and it should pass when the Flash Attention is turned off ([`torch.backends.cuda.enable_flash_sdp`](https://pytorch.org/docs/stable/backends.html#torch.backends.cuda.enable_flash_sdp)). `fused_sdp_choice` which nvFuser's checker uses would pick up the value of this flag
https://github.com/Lightning-AI/lightning-thunder/blob/947e9ef2eca5c1a1b3c380d38e16c0afd7533b06/thunder/executors/utils.py#L216
https://github.com/Lightning-AI/lightning-thunder/blob/947e9ef2eca5c1a1b3c380d38e16c0afd7533b06/thunder/executors/nvfuserex_impl.py#L2456

Reply:
Thanks @IvanYashchuk  for the fix.